addons/account/models/account_report.py

```diff
@@ -253,6 +253,12 @@ def copy(self, default=None):
                 for old_code, new_code in code_mapping.items():
                     copied_formula = re.sub(f"(?<=\\W){old_code}(?=\\W)", new_code, copied_formula)
                 expression.formula = copied_formula.strip() # Remove the spaces introduced for lookahead/lookbehind
+                # Repeat the same logic for the subformula, if it is set.
+                if expression.subformula:
+                    copied_subformula = f" {expression.subformula} "
+                    for old_code, new_code in code_mapping.items():
+                        copied_subformula = re.sub(f"(?<=\\W){old_code}(?=\\W)", new_code, copied_subformula)
+                    expression.subformula = copied_subformula.strip()
```

Comment:
Maybe we should add a test for this ? :thinking:  I don't know if there is already one for report copy at all, but it would a nice add IMO

Reply:
I didn't see one in both the account & account_reports modules.

I added a test in account (to avoid needing to open another PR) let me know if it's fine by you